pkg/v3/plugin/ocr3_test.go

```diff
@@ -13,14 +13,15 @@ import (
 	"testing"
 
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
-	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 	"github.com/stretchr/testify/assert"
 
 	ocr2keepers2 "github.com/smartcontractkit/chainlink-automation/pkg/v3"
 	"github.com/smartcontractkit/chainlink-automation/pkg/v3/plugin/hooks"
 	"github.com/smartcontractkit/chainlink-automation/pkg/v3/service"
-	ocr2keepers "github.com/smartcontractkit/chainlink-automation/pkg/v3/types"
+	types2 "github.com/smartcontractkit/chainlink-automation/pkg/v3/types"
```

Comment:
Let's use a better name than `types2`, maybe we can rename offchainreporting2plus/types to `ocr2plustypes` and keep this one as `types`